core/src/main/scala/io/finch/internal/Accept.scala

```diff
@@ -0,0 +1,21 @@
+package io.finch.internal
+
+import com.twitter.util.Try
+import javax.activation.MimeType
+
+case class Accept(`type`: String, subtype: String) { self =>
+
+  def matches(other: Accept): Boolean = {
```

Comment:
Should we define `Eq[Accept]` instance instead?

Reply:
Yep, I like this idea